README.md

```diff
@@ -38,6 +38,8 @@ Questions are grouped into three buckets: [General](#general), [Technical](#tech
     + What kind of elements can `title` attributes be used on?
     + What sort of information is appropriate for use with the `title` attribute?
 - Describe a scenario where you might need to use `aria-describedby`.
+- What is "tabtrapping" and when should it be used?
+- What are different use cases and values for `tabindex`?
```

Comment:
suggest rewording to:
"Describe a situation where one might need to add or modify the the focusability of an element by using the `tabindex` attribute."

Reply:
awesome; updated